srsenb/sib.conf.example

```diff
@@ -1,3 +1,12 @@
+#####################################################################
+# sib1 configuration options (See TS 36.331)
+#
+# additional_plmns: A list of additional PLMN identities.
+#     mcc: MCC
```

Comment:
how would I specify multiple PLMNs? Can you give an example? Can I select the `reserved` option per PLMN?

Reply:
It would look like:
```
sib1 =
{
    intra_freq_reselection = "Allowed";
    q_rx_lev_min = -65;
    //p_max = 3;
    cell_barred = "NotBarred"
    si_window_length = 20;
    sched_info =
    (
        {
            si_periodicity = 16;
            // comma-separated array of SIB-indexes (from 3 to 13), leave empty or commented to just scheduler sib2
            si_mapping_info = [ 3 ];
        }
    );
    system_info_value_tag = 0;
    additional_plmns = 
    (
        {
            mcc = "345";
            mnc = "67";
        }
    );
};
```
And you can select the `reserved` option by adding `cell_reserved_for_oper = "reserved";`, but the default is `notReserved`